### PR TITLE
PAF-50 Deploy ims-resolver

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
## What

Deploy ims resolver as part of PRs

## Why

Because we need the ims-resolver to send the form data and docs to the IMS system

### CODE REVIEW

File: kube/ims-resolver/ims-resolver-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            - name: IMS_API_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: ims-credentials
+                  key: ims_password
+            - name: WSDL
+              valueFrom:
+                secretKeyRef:
+                  name: ims-credentials
+                  key: ims_user
```
Comment (shamiluwais) [inline kube/ims-resolver/ims-resolver-deploy.yml:69]: Should be wsdl

File: kube/ims-resolver/ims-resolver-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            - name: IMS_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: ims-credentials
+                  key: ims_password
```
Comment (shamiluwais) [inline kube/ims-resolver/ims-resolver-deploy.yml:74]: Should be key: ims_endpoint?

File: kube/ims-resolver/ims-resolver-deploy.yml (hunk 1 above)
Comment (sulthan-ahmed) [reply kube/ims-resolver/ims-resolver-deploy.yml]: sorted

File: kube/ims-resolver/ims-resolver-deploy.yml (hunk 2 above)
Comment (sulthan-ahmed) [reply kube/ims-resolver/ims-resolver-deploy.yml]: sorted

**Comment by sulthan-ahmed** [comment]
are you happy to approve this now? @shamiluwais

**Review by shamiluwais — APPROVED** [review]
lgtm

**Comment by sulthan-ahmed** [comment]
Thanks @shamiluwais

**Pull request merged**